src/components/schedule-visualizer/robot-trajectory.tsx

```diff
@@ -71,7 +74,7 @@ export const RobotTrajectory = React.memo(
         case TrajectoryColor.Shades:
```

Comment:
It seems that the orange color will only appear when the shades option is selected. Can we implement it for others options too?

![image](https://user-images.githubusercontent.com/11761240/93805095-3ad87b00-fc15-11ea-9c9f-6c1556728506.png)

Reply:
apply to all modes a895dae